config.js

```diff
@@ -0,0 +1 @@
+module.exports='YOUR HOST NAME HERE'
```

Comment:
this should probably be 
module.exports = {
  'hostname': ''
}

else this file can have only hostname config